### PR TITLE
Update readme with correct HC version number

### PR DESCRIPTION
**Pull request by pdaoust**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,9 +63,9 @@ The next time you enter the dev shell, the updated binaries will be downloaded a
 
 ## Including other versions of Holochain
 
-By default the Holonix input references the `main` branch of the Holonix repository. Whatever the current revision of that branch at the moment of invoking the dev shell for the first time is will be written to the `flake.lock` file. The `main` branch uses version 0.4 of Holochain and compatible versions of scaffolding etc.
+By default the Holonix input references the `main` branch of the Holonix repository. Whatever the current revision of that branch at the moment of invoking the dev shell for the first time is will be written to the `flake.lock` file. The `main` branch uses version 0.5 of Holochain and compatible versions of scaffolding etc.
 
-If you want to develop using the 0.3 branch of Holochain instead, you can create the corresponding Nix flake running this command inside your project folder:
+If you want to develop using the 0.3 or 0.4 branch of Holochain instead, you can create the corresponding Nix flake running this command inside your project folder:
```
Comment (ThetaSinner) [inline README.md]: This is not accurate against the command below. It specifically gives you the ~~0.4~~ 0.3 version. I think until 0.4 is released we should keep recommending 0.3 here

Comment (pdaoust) [reply README.md]: I don't quite understand -- you're saying the command below specifically gives you 0.4?

Comment (pdaoust) [reply README.md]:
```suggestion
If you want to develop using the 0.3 (recommended) or 0.4 (RC) release series of Holochain instead, you can create the corresponding Nix flake by running this command inside your project folder, replacing `<ver>` with the correct version number:
```

Comment (ThetaSinner) [reply README.md]: The top part is correct, not the "0.3 or 0.4" part

File: README.md [hunk 2]
```diff
@@ -63,9 +63,9 @@ The next time you enter the dev shell, the updated binaries will be downloaded a
...
 ## Including other versions of Holochain
 
-By default the Holonix input references the `main` branch of the Holonix repository. Whatever the current revision of that branch at the moment of invoking the dev shell for the first time is will be written to the `flake.lock` file. The `main` branch uses version 0.4 of Holochain and compatible versions of scaffolding etc.
+By default the Holonix input references the `main` branch of the Holonix repository. Whatever the current revision of that branch at the moment of invoking the dev shell for the first time is will be written to the `flake.lock` file. The `main` branch uses version 0.5 of Holochain and compatible versions of scaffolding etc.
 
-If you want to develop using the 0.3 branch of Holochain instead, you can create the corresponding Nix flake running this command inside your project folder:
+If you want to develop using the 0.3 (recommended) or 0.4 (RC) release series of Holochain instead, you can create the corresponding Nix flake by running this command inside your project folder, replacing `<ver>` with the correct version number:
 
 ```shell
 nix flake init -t github:holochain/holonix?ref=main-0.3
```
Comment (pdaoust) [inline README.md]:
```suggestion
nix flake init -t github:holochain/holonix?ref=main-<ver>
```

**Review by ThetaSinner — APPROVED** [review]

**Pull request merged**